client/app/components/sections/ManageAvailability/ManageAvailability.css

```diff
@@ -1,3 +1,26 @@
+@keyframes stroke {
```

Comment:
Are these in global namespace (or how these work within this file)?
(If they are in global namespace, maybe a bit more complex names would be better)